src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1050,7 +1050,7 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
     <Field Name="LocalSymbol" Type="LocalSymbol"/>
-    <Field Name="IsDeclaration" Type="bool"/>
+    <Field Name="IsDeclaration" Type="IsDeclaration" Null="allow"/>
```

Comment:
>IsDeclaration [](start = 38, length = 13)

I think all non-intrinsic value types should be listed at the beginning of the root node.

Reply:
Are you saying I should move this line (with `DeclarationKind`) above the line with `Type`? 
If so, I don't understand what difference that makes. Is that for readability, for packing bits, or some other reason?

---
In reply to: [191559493](https://github.com/dotnet/roslyn/pull/26828#discussion_r191559493) [](ancestors = 191559493)